test/e2e/e2e-utils.ts

```diff
@@ -5,11 +5,10 @@ import { back as nockBack, BackMode, Definition } from 'nock';
 
 export function decodeBuffer(recordingResponse: Definition['response']) {
   try {
-    // Decode the hex buffer that nock made
-    const response = Array.isArray(recordingResponse)
-      ? recordingResponse.join('')
-      : recordingResponse?.toString() || '';
-    const decoded = Buffer.from(response, 'hex');
+    const isArray = Array.isArray(recordingResponse);
+    if (!isArray) return recordingResponse; // Some Auth0 Management API endpoints aren't gzipped, we can tell this if they are an array or not
```

Comment:
From my tests, there appear to be some Management API endpoints that do not gzip their payloads, so when attempting to decode for the purposes of recording, it would fail. I found that this check if the payload is an array is a simplistic way of determining wether to decode the payload. If it is not an array of hex bytes, then it its likely already a parsed payload. This check is probably imperfect and susceptible to edge cases, but it works for now.